.github/workflows/nix.yml

```diff
@@ -26,3 +23,41 @@ jobs:
 
       - name: Build all flake outputs
         run: om ci
+
+      - name: Build and push Docker image
+        if: github.event_name == 'push' && github.event.pull_request.merged == true
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          set -euo pipefail
+
+          ARG1="${1:-dockerImage}"
```

Comment:
Just inline `ARG1`; this is not a script that someone will invoke passing arguments anyway.